src/platform/packages/shared/content-management/table_list_view_table/src/components/use_tag_filter_panel.tsx

```diff
@@ -48,23 +48,12 @@ export const useTagFilterPanel = ({
   addOrRemoveIncludeTagFilter,
 }: Params) => {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
-  // When the panel is "in use" it means that it is opened and the user is interacting with it.
-  // When the user clicks on a tag to select it, the component is unmounted and mounted immediately, which
-  // creates a new EUI transition "IN" which makes the UI "flicker". To avoid that we pass this
-  // "isInUse" state which disable the transition.
-  const [isInUse, setIsInUse] = useState(false);
```

Comment:
Unrelated to the failure, this hack wasn't needed since the refactor to use panel context, so cleaning it up